OpenSim/Simulation/Model/ModelVisualizer.h

```diff
@@ -181,9 +181,10 @@ class OSIMSIMULATION_API ModelVisualizer {
     No attempt is made to validate the contents of the file or whether it
     has a supported extension; we're just looking for a file of the given
     name that exists and is readable. **/
-    bool findGeometryFile(const std::string&          geoFile,
+    static bool findGeometryFile(const Model& aModel,
+                          const std::string&          geoFile,
                           bool&                       isAbsolute,
-                          SimTK::Array_<std::string>& attempts) const;
+                          SimTK::Array_<std::string>& attempts);
```

Comment:
A method whose name begins with `find` should be `const`. Something is wrong if it has to be non-const.